### DISCUSSION TITLE
make a dock widget

### DISCUSSION THREAD

**Discussion by ddkwork** [cogentcore/core#742] [General]
![screenshots](https://github.com/goki/gi/assets/19886504/f2c81521-b53f-4dcd-b62e-dbb43196b96d)
![screenshots](https://github.com/goki/gi/assets/19886504/58582359-d92b-4728-98c0-c87569f57abe)

**Comment by kkoreilly** [comment]
I will look into this more soon.

**Comment by ddkwork** [comment]
This may only require extending the tab property, so turn off https://github.com/cogentcore/core/issues/863

**Comment by runrc** [comment]
This functionality would be immensely useful!

**Reply by kkoreilly** [reply]
I just filed #993 for this; if you have any feedback on how this should look, you can add it there.
